CONTRIBUTING.md

```diff
@@ -144,8 +144,8 @@ specifying the environment variable `BUILD_PARALLEL=<n>`.
 
 ### Building against different CUDA Toolkit versions
 
-You can build against different versions of the CUDA Toolkit by using one of the following profiles:
-* `-Pcuda11` (CUDA 11.0/11.1/11.2, default)
+You can build against different versions of the CUDA Toolkit by modifying the variable `cuda.version`:
+* `-Dcuda.version=cuda11` (CUDA 11.0/11.1/11.2, default)
```

Comment:
nit,
```suggestion
* `-Dcuda.version=cuda11` (set to  `cuda11` for CUDA 11.x, or `cuda12` for CUDA 12.x)
```

Reply:
cuda12 should be on a separate line.  This already has the list-of-1 problem, so might as well lean into the fact that it's a list of values and add a new entry.